libraries/AP_AHRS/AP_AHRS.h

```diff
@@ -649,6 +661,8 @@ class AP_AHRS
         uint8_t fly_forward             : 1;    // 1 if we can assume the aircraft will be flying forward on its X axis
         uint8_t correct_centrifugal     : 1;    // 1 if we should correct for centrifugal forces (allows arducopter to turn this off when motors are disarmed)
         uint8_t wind_estimation         : 1;    // 1 if we should do wind estimation
+        uint8_t takeoff_expected        : 1;    // 1 if the vehicle is in a state that takeoff might be expected.  Ground effect may be in play.
+        uint8_t touchdown_expected       : 1;    // 1 if the vehicle is in a state that touchdown might be expected.  Ground effect may be in play.
```

Comment:
nitpick: there's a 1 space error in whitespace before the comment.

Reply:
fixed